apps/emqx_management/test/emqx_mgmt_api_test_util.erl

```diff
@@ -109,10 +139,7 @@ do_request_api(Method, Request, Opts) ->
     end.
 
 auth_header_() ->
```

Comment:
nit:
can we try to rename this API?
even auth_header2 is better than auth_header_

Reply:
There are three duplicated HTTP test helper files, and there are many same functions in these files, I plan to unite them into one to reduce the chaos, so this name can be renamed in that PR